src/App.tsx

```diff
@@ -1,24 +1,200 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorModal } from './components/ErrorModal/ErrorModal';
+import { LoadContext } from './Context/LoadContext';
 import { UserWarning } from './UserWarning';
+import { FilterType, ErrorType } from './typedefs';
+import { Todo } from './components/TodoItem/Todo';
+import {
+  addTodo,
+  getTodos,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+import { getFilteredTodos } from './helpers/helpers';
+import { useFilteredTodos } from './hooks/useFilteredTodos';
 
-const USER_ID = 0;
+const USER_ID = 6748;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.ALL);
+  const [error, setError] = useState<ErrorType>(ErrorType.NONE);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [unableField, setUnableField] = useState(false);
+  const [loadingTodos, setLoadingTodos] = useState([0]);
+
+  const {
+    activeTodos,
+    completedTodos,
+  } = useFilteredTodos({ todos });
+
+  const showError = (errorType: ErrorType) => {
+    setError(errorType);
+    setTimeout(() => setError(ErrorType.NONE), 3000);
+  };
```

Comment:
Just a suggestion, we can move this function with error state to separate hook that will manage errors (have state with it, reset it after 3 seconds) and return errorType and function to set it

Reply:
Done)